app/components/round-resource.js

```diff
@@ -14,8 +14,12 @@ export default Component.extend({
     }
   }),
 
-  score: computed('users.user.id', 'scorecards.user.id', function() {
-    return this.get('scorecards').findBy('user.id', 4);
+  score: computed('round.scorecards.[]', function() {
+    return this.get('round.scorecards.firstObject.score');//.findBy('user.id', 4);
```

Comment:
Ah sweet. Okay I see where this is going now. That first commit had me confused.